io.github.mpc_qt.mpc-qt.yml

```diff
@@ -0,0 +1,294 @@
+                      timestamp-query: .published_at
+
+          - name: libplacebo
+            buildsystem: meson
+            config-opts:
+              - -Dopengl=enabled
+              - -Dvulkan=enabled
+              - -Dglslang=disabled
+              - -Dd3d11=disabled
+              - -Ddemos=False
+            sources:
+              - type: archive
+                url: https://code.videolan.org/videolan/libplacebo/-/archive/v7.349.0/libplacebo-v7.349.0.tar.gz
+                sha256: 79120e685a1836344b51b13b6a5661622486a84e4d4a35f6c8d01679a20fbc86
+                x-checker-data:
+                  type: anitya
+                  project-id: 20101
+                  stable-only: true
+                  url-template: https://code.videolan.org/videolan/libplacebo/-/archive/v$version/libplacebo-v$version.tar.gz
+              - type: archive
+                url: https://github.com/pallets/jinja/archive/refs/tags/3.1.5.tar.gz
+                sha256: 684d78790cc747f4b4b68b365783af8cac770c2c66d9f414b21c8b8480e019d0
+                dest: 3rdparty/jinja
+                x-checker-data:
+                  type: anitya
+                  project-id: 3894
+                  url-template: https://github.com/pallets/jinja/archive/refs/tags/$version.tar.gz
+              - type: archive
+                url: https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.8.tar.gz
+                sha256: 44f06f9195427c7017f5028d0894f57eb216b0a8f7c4eda7ce883732aeb2d0fc
+                dest: 3rdparty/glad
+                x-checker-data:
+                  type: anitya
+                  project-id: 300234
+                  url-template: https://github.com/Dav1dde/glad/archive/refs/tags/v$version.tar.gz
+
+          - name: libXpresent
+            buildsystem: autotools
+            sources:
+              - type: archive
+                url: https://xorg.freedesktop.org/archive/individual/lib/libXpresent-1.0.1.tar.xz
+                sha256: b964df9e5a066daa5e08d2dc82692c57ca27d00b8cc257e8e960c9f1cf26231b
+                x-checker-data:
+                  type: anitya
+                  project-id: 17166
+                  stable-only: true
+                  url-template: https://xorg.freedesktop.org/archive/individual/lib/libXpresent-$version.tar.xz
+
+          - name: nv-codec-headers
+            cleanup:
+              - '*'
+            no-autogen: true
+            make-install-args:
+              - PREFIX=/app
+            sources:
+              - type: archive
+                url: https://github.com/FFmpeg/nv-codec-headers/archive/refs/tags/n13.0.19.0.tar.gz
+                sha256: 86d15d1a7c0ac73a0eafdfc57bebfeba7da8264595bf531cf4d8db1c22940116
+                x-checker-data:
+                  type: anitya
+                  project-id: 223796
+                  stable-only: true
+                  url-template: https://github.com/FFmpeg/nv-codec-headers/archive/refs/tags/n$version.tar.gz
+
+          - name: ffmpeg
```

Comment:
why not using the ffmpeg from the runtime?

Reply:
h265 didn't work and h264 didn't have hardware accelerated decoding.

Reply:
you need the ffmpeg-full extension it has h265 support.

Reply:
this exact version of ffmpeg is in the runtime and ffmpeg-full in 6.8